src/aiperf/records/records_manager.py

```
@@ -725,12 +732,25 @@ async def _maybe_trigger_failed_request_abort(self, phase: CreditPhase) -> None:
+                service_id=self.service_id,
+                reason=ProfileCancelReason.FAILED_REQUEST_THRESHOLD,
+                reason_detail=(
+                    f"{error_records}/{total} profiling requests failed "
+                    f"({rate:.1%}), exceeding the "
+                    f"--failed-request-threshold limit of "
+                    f"{self._failed_request_threshold:.1%}. "
+                    "Check inference server logs."
+                ),
+            )
+            await self.publish(command)
         except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)